class-gf-razorpay.php

```diff
@@ -163,6 +174,45 @@ public function plugin_settings_fields()
                         'type'        => 'text',
                         'class'       => 'medium',
                     ),
+                    array(
+                        'name'   => self::GF_RAZORPAY_PAYMENT_ACTION,
+                        'label' => esc_html__('Payment Action', 'razorpay'),
+                        'tooltip' => esc_html__('Payment action on order compelete.', $this->_slug),
```

Comment:
Instead of tooltip you can use description to show the content here.

Reply:
added for webhook URL, other is fine with tooltip.